From 96f6ef0c27ae46b61cd1165384aed56155a9a373 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Thu, 2 Mar 2017 10:00:15 +0000
Subject: [PATCH 1022/1023] mach-imx: gpio-label: Add gpio labels for new GPIOs

The revision A1 of the hardware introduces several new GPIOs which need labels

Ticket#1668
---
 arch/arm/mach-imx/gpio_label.h | 31 ++++++++++++++++++++-----------
 1 file changed, 20 insertions(+), 11 deletions(-)

diff --git a/arch/arm/mach-imx/gpio_label.h b/arch/arm/mach-imx/gpio_label.h
index 9468d98..879836d 100644
--- a/arch/arm/mach-imx/gpio_label.h
+++ b/arch/arm/mach-imx/gpio_label.h
@@ -1,17 +1,20 @@
 #ifndef __GPIO_LABEL_H__
 #define __GPIO_LABEL_H__
 
-#define CAN1_EN "can1_en"
-#define CAN2_EN "can2_en"
-#define CAN1_STBY "can1_stby"
-#define CAN2_STBY "can2_stby"
-#define CAN1_TERM "can1_term"
-#define CAN2_TERM "can2_term"
+#define CAN1_EN		"can1_en"
+#define CAN2_EN		"can2_en"
+#define CAN1_STBY	"can1_stby"
+#define CAN2_STBY	"can2_stby"
+#define CAN1_TERM	"can1_term"
+#define CAN2_TERM	"can2_term"
+#define CAN2_WAKE   "can2_wakeup"
 
-#define V5P_E 		"5p_e"
-#define V5P_GOOD 	"5p_good"
+#define V5P_EXT_E 	"5p_ext_e"
+#define V5P_ISO_E 	"5p_iso_e"
 #define WIFI_PWR_E	"wifi_pwr_e"
 #define GSM_PWR_E	"gsm_pwr_e"
+#define GSM_E_RST   "gsm_e_rst"
+#define GSM_IGN     "gsm_ign"
 #define SIM_SEL		"sim_sel"
 #define GNSS_PWR_E	"gnss_pwr_e"
 #define K_L_PU_E 	"k_l_pu_e"
@@ -26,17 +29,23 @@
 #define CHRG_IRQ	"charge_irq"
 #define TEMP_IRQ	"temp_irq"
 #define BUZZER      "buzzer"
+#define LIN_E       "lin_e"
+#define HOST2BT     "host2bt"
+#define HOST2WL     "host2wl"
+#define OTG_DRVBUS  "otg_drvbus"
 
+#define DI1			"di1"
 #define DI2			"di2"
 #define DI3			"di3"
 #define DI_K15		"di_k15"
 #define DI_K30		"di_k30"
 
-#define DO1_OUT	"do1_out"
+#define DO1_OUT		"do1_out"
 #define DO1_STATE	"do1_state"
-#define DO2_OUT	"do2_out"
+#define DO2_OUT		"do2_out"
 #define DO2_STATE	"do2_state"
-#define DO3_OUT	"do3_out"
+#define DO3_OUT		"do3_out"
 #define DO3_STATE	"do3_state"
+#define DO_IRQ      "do_irq"
 
 #endif // __GPIO_LABEL_H__
-- 
2.1.4

